repository/n.json

```diff
@@ -34,6 +34,18 @@
 				}
 			]
 		},
+		{
+			"name": "Nano Bots",
+			"details": "https://github.com/icebaker/sublime-nano-bots",
+			"issues": "https://github.com/icebaker/sublime-nano-bots/issues",
```

Comment:
Since this is the default location for issues you can leave it out, package control will figure that out for you.